Jamulus.pro

```diff
@@ -81,6 +81,7 @@ win32 {
     DEFINES += NOMINMAX # solves a compiler error in qdatetime.h (Qt5)
     RC_FILE = src/res/win-mainicon.rc
     mingw* {
+        DEFINES += _WIN32_WINNT=0x0600 # solves missing inet_pton in CSocket::SendPacket
```

Comment:
Is this fixed with the Qt6 creator? If that's the case, I'd add it here.

Reply:
It all got too complicated to try...  Ideally, both should work together.